br/pkg/aws/ebs.go

```diff
@@ -149,8 +149,18 @@ func (e *EC2Session) CreateSnapshots(backupInfo *config.EBSBasedBRMeta) (map[str
 				resp, err := e.createSnapshotsWithRetry(context.TODO(), &createSnapshotInput)
 
 				if err != nil {
-					return errors.Trace(err)
+					if aErr, ok := err.(awserr.Error); !ok || aErr.Code() != "TagLimitExceeded" {
```

Comment:
why not check the error in the ec2 api call? In the `createSnapshotsWithRetry` function, it will do useless retry if the error is `TagLimitExceeded`